h2o-core/src/main/java/water/util/PojoUtils.java

```diff
@@ -524,8 +524,10 @@ else if (f.getType() == float.class && (value instanceof Number))
           Enum[] valuesCast = (Enum[]) Array.newInstance(f.getType().getComponentType(), valuesTyped.length);
           for (int i = 0; i < valuesTyped.length; i++) {
             String v = (String) valuesTyped[i];
-            Enum enu = EnumUtils.valueOfIgnoreCase((Class<Enum>)f.getType().getComponentType(), v);
-            if (enu == null) throw new IllegalArgumentException("Field = " + fieldName + " element cannot be set to value = " + value);
+            final IllegalArgumentException fieldNotSettablException = new IllegalArgumentException("Field = " + fieldName + " element cannot be set to value = " + value);
```

Comment:
personally I am against instantiating exception when they are not really needed, I would move this inside the orElseThrow (same below)

Reply:
I'd need value to be final then, so I'll make a final copy of `value` locally.

Reply:
Luckily, there is `final Object[] valuesTyped = ((Object[]) value);`  just above, which can be made final. And as it's an array of Objects, I've used `Arrays.toString(valuesTyped)` at the end of the exception's message properly.